main/ejml-simple/src/org/ejml/simple/AutomaticSimpleMatrixConvert.java

```diff
@@ -35,7 +34,7 @@ public void specify0( SimpleBase a , SimpleBase ...inputs ) {
         SimpleBase array[] = new SimpleBase[inputs.length+1];
         System.arraycopy(inputs,0,array,0,inputs.length);
         array[inputs.length] = a;
-        specify(inputs);
+        specify(array);
```

Comment:
well that was a silly type-o. Did all the work then passed in the wrong object. Thanks for the fix!